core/conversion/conversionctx/ConversionCtx.cpp

```diff
@@ -64,15 +63,7 @@ ConversionCtx::ConversionCtx(BuilderSettings build_settings)
         cfg->setFlag(nvinfer1::BuilderFlag::kFP16);
         break;
       case nvinfer1::DataType::kINT8:
-        TORCHTRT_CHECK(
```

Comment:
I dont think we need this warning unless they are trying to use anything related to the calibrator argument. We can just assume explicit quantization

Reply:
Could maybe set it as a debug message if you want it to print out